templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -512,8 +512,10 @@ export class {{ service.name }}Client {
 {%- for method in service.simpleMethods %}
 /**
 {{- util.printComments(method, service) }}
+ {%- if not api.handwrittenLayer %}
```

Comment:
how is the `api.handwrittenLayer` variable set?

Reply:
I remember the variable set through bazel rules ([example](https://github.com/googleapis/googleapis/blob/master/google/bigtable/v2/BUILD.bazel#L220))

Please correct me @sofisl if I am wrong.

Reply:
Yep, exactly! We haven't added it yet, but it would be set there, where Summer says ^